lfucg-exactions/server/accounts/utils.py

```diff
@@ -35,4 +35,44 @@ def calculate_account_balance(account_id):
     current_account_balance = to_value - from_value - payment_value
 
     return current_account_balance
+
+def calculate_agreement_balance(agreement_id):
+    agreement = Agreement.objects.filter(id=agreement_id)
+
+    ledger_from_value = 0
+    ledger_to_value = 0
+    
+    related_account_id = agreement[0].account_id.id
+
+    account_ledger_agreements_from = AccountLedger.objects.filter(agreement=agreement_id, account_from=related_account_id)
+    account_ledger_agreements_to = AccountLedger.objects.filter(agreement=agreement_id, account_to=related_account_id)
+
+    if account_ledger_agreements_from.exists():
+        for ledger_from in account_ledger_agreements_from:
+            ledger_from_credits = ledger_from.non_sewer_credits + ledger_from.sewer_credits
+            ledger_from_value += ledger_from_credits
```

Comment:
ledger_from_credits is not used anywhere else. can be simplified to ledger_from_value += ledger_from.non_sewer_credits + ledger_from.sewer_credits